go/src/koding/klient/machine/index/change.go

```diff
@@ -103,6 +103,25 @@ func (cm *ChangeMeta) Coalesce(newer ChangeMeta) ChangeMeta {
 	}
 }
 
+// String implements fmt.Stringer interface and pretty prints stored change.
+func (cm *ChangeMeta) String() string {
+	cpy := atomic.LoadUint64((*uint64)(cm))
+
+	const cmstr = "LRaduH"
```

Comment:
using heap-allocated map just to print a small value. This is not a big overhead but this solution is as simple as map and much faster. This is the only reason and I'm not opposed to using maps(or I just read std lib too much:) )